{"name": "test_log_in_failed", "status": "passed", "attachments": [{"name": "log", "source": "3bd98cc1-c221-413b-a476-a039e2859e1f-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "e229e680-4015-4269-ac14-9d4209733563-attachment.txt", "type": "text/plain"}], "start": 1694715424079, "stop": 1694715442201, "uuid": "22fbc5b1-c7c7-43fb-8457-9a9db59e5e0e", "historyId": "f000e49c996cfb0a75f91c6c9a56e1a2", "testCaseId": "f660f1d514dce5eb4c33a4e6c2d5d52e", "fullName": "page_object_pattern.tests.login_test.TestLogIn#test_log_in_failed", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "page_object_pattern.tests"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestLogIn"}, {"name": "host", "value": "Piotr"}, {"name": "thread", "value": "14340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "page_object_pattern.tests.login_test"}]}